man/resolved.conf.xml

```diff
@@ -227,10 +227,11 @@
 
       <varlistentry>
         <term><varname>Cache=</varname></term>
-        <listitem><para>Takes a boolean argument. If <literal>yes</literal> (the default), resolving a domain name
+        <listitem><para>Takes (yes, no, no-negative) as argument. If <literal>yes</literal> (the default), resolving a domain name
```

Comment:
`Takes a boolean or <literal>no-negative</literal> as argument.`
You could also change the existing text to use `If true` instead of `If <literal>yes</literal>`, if you want to.

Reply:
Done, thanks for the pointers.